From f29125186dc3a35a958aa2611bf542a079a67010 Mon Sep 17 00:00:00 2001
From: Robin Gong <b38343@freescale.com>
Date: Thu, 25 Jun 2015 18:38:10 +0800
Subject: [PATCH] MLK-11172 ARM: imx: pm-imx7: correct uart node path

Since SPBA node which is the parent of uart was added in commit
e0c624ac68d6749bf760da6e6c05e5dd2165b20c, uart node path should be corrected,
otherwise console can't work after system resume back with mega/fast off.

Signed-off-by: Robin Gong <b38343@freescale.com>
---
 arch/arm/mach-imx/pm-imx7.c | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/arch/arm/mach-imx/pm-imx7.c b/arch/arm/mach-imx/pm-imx7.c
index 105aff0..caf1dde 100644
--- a/arch/arm/mach-imx/pm-imx7.c
+++ b/arch/arm/mach-imx/pm-imx7.c
@@ -620,7 +620,7 @@ void __init imx7d_pm_init(void)
 	WARN_ON(!ocram_saved_in_ddr);
 
 	np = of_find_node_by_path(
-		"/soc/aips-bus@30800000/serial@30860000");
+		"/soc/aips-bus@30800000/spba-bus@30800000/serial@30860000");
 	if (np)
 		console_base = of_iomap(np, 0);
 }
-- 
1.8.0

